### ISSUE TITLE
New presto driver timeout

### ISSUE THREAD

**Issue by sananguliyev** [metabase/metabase#27731]
I have a Trino cluster behind the load balancer and for using TLS support from Trino. 

I can connect to my cluster from trino cli with no problem, but when I try the new presto driver from metabase it throws timeout error. Furthermore, I have tried the same configuration with depreciated driver, and it works perfectly, too. 

My question is what is the main difference between old and new drivers which causes timeour issue.

Thanks in advance

**Comment by paoliniluis** [comment]
Hi, have you tried using the Starburst driver? https://github.com/starburstdata/metabase-driver/

**Comment by sananguliyev** [comment]
Hi @paoliniluis 

Thanks for your answer. No, I have not tried it because of the built-in driver. As far as I understand metabase presto driver supports Trino what I did not get is why the depreciated driver works, but new one give timeout error which I have no error logs or something on Trino cluster.

I just want to figure out whether the problem is on my cluster or metabase. If it's on metabase side, then fixing it would be a better solution, so none will have same issue again.

**Comment by paoliniluis** [comment]
Presto != Trino, that's the why :)

**Comment by sananguliyev** [comment]
I know the difference, history and all this stuff. It's not the answer. I just call it presto because how it's written in dropdown menu in metabase. I did not mention PrestoDB. I specifically said Presto because I did not check the code. Maybe they try both `presto` and `trino` in order to have one driver for both since they are same anyway. 

I did not understand whether your answer is sarcasm or not but the reason of question is contribution not saying solve my problem. It's fine for me if metabase behave like that with community. 

Anyway, thanks a lot for trying to write something.
